src/components/Footer/Footer.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import './Footer.scss';
+
+class Footer extends React.Component {
+  render() {
+    return (
+      <div className="footerContainer">
```

Comment:
- footer tag가 더 적합하지 않을까요?
- 최상위 태그의 className의 경우에는 네스팅을 해서 다른 컴포넌트에 영향을 안미치기 위해서 유니크할 수 있게 컴포넌트의 이름과 동일하게 지어주는게 위험성이 적습니다 
```suggestion
      <div className="Footer">
```

Reply:
이부분도 반영하였습니다! 앞으로 컴포넌트 만들때도 계속 적용하도록 하겠습니다 🙆‍♀️